modcc/io/bulkio.hpp

```diff
@@ -0,0 +1,49 @@
+#pragma once
+
+// Read or write the contents of a file in toto.
+
+#include <string>
+#include <iterator>
+#include <fstream>
+
+namespace io {
+
+template <typename Assignable>
```

Comment:
It looks like `Assignable` is a container. I see that your only requirement here is that `A` provides an `assign()` function, so perhaps you don't need another concept name, e.g., `SequenceContainer` etc. Would it make sense to have sth like `AssignableContainer`, 'cause initially I got a bit confused. I thought you need sth to support assignment and there are already a `CopyAssignable` and `MoveAssignable` concepts with a different meaning.

Reply:
Certainly happy to rename! Perhaps `HasAssign`?

Reply:
`HasAssign` sounds fine for me.